news/2024/2024-02-02-osumania-7k-world-cup-2024-round-of-16-recap.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2024: Round of 16 Recap"
+date: 2024-02-02 12:00:00 +0000
+---
+
+The cogs of competition keep on turning! Who's going to be able to keep that momentum running in rounds to come?
+
+![](/wiki/Tournaments/MWC/2024_7K/img/mwc7k2024-banner.jpg)
+
+The stakes for all teams start becoming higher as more rounds go by. As the mappools grow ever so slightly more difficult and the looming peril of dropping to the losers' bracket stalks players, our contestants' wits are only getting stronger!
+
+## Mappool showcase
+
+[**Download the pool here! (50 MB)**](https://osu.ppy.sh/beatmaps/packs/P234)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=v2039649748&parent=osu.ppy.sh&autoplay=false&t=0h36m41s" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
```

Comment:
wrong timestamp?

Reply:
Correct, should practically start at the beginning of the VOD, so the timestamp is not necessary.